sdk/trace/src/main/java/io/opentelemetry/sdk/trace/SdkTracerProviderBuilder.java

```diff
@@ -119,6 +120,22 @@ public SdkTracerProviderBuilder setSampler(Sampler sampler) {
     return this;
   }
 
+  /**
+   * Control the registration of a shutdown hook to shut down the SDK when appropriate.
+   *
+   * <p>Skipping the registration of the shutdown hook may cause unexpected behavior. This
+   * configuration is for SDK consumers that require control over the SDK lifecycle. In this case,
+   * alternatives must be provided by the SDK consumer to shut down the SDK.
+   *
+   * @param registerShutdownHook a boolean <code>true</code> will register the hook, otherwise
+   *     <code>false</code> will skip registration.
+   * @return this
+   */
+  public SdkTracerProviderBuilder setRegisterShutdownHook(boolean registerShutdownHook) {
```

Comment:
I'd prefer this were just `registerShutdownHook` without the `set`. We have no reason we need to adhere to the very antiquated javabean spec here.